app/src/main/java/org/glucosio/android/tools/GlucosioConverter.java

```diff
@@ -53,14 +53,12 @@ public static double a1cToGlucose(double a1c) {
         return round((a1c * 28.7) - 46.7, 2);
     }
 
-    public static int kgToLb(int kg) {
-        Double d = kg * 2.20462;
-        return d.intValue();
+    public static double kgToLb(double kg) {
+        return kg * 2.20462;
```

Comment:
`create a constant so it make's sense? I have no idea what is 18 atm`
same for above an below

Reply:
Also extracted constant